themes/bootstrap5/templates/header.phtml

```diff
@@ -1,117 +1,125 @@
-<?php $account = $this->auth()->getManager(); ?>
+<?php
+/** @var \Laminas\View\Renderer\PhpRenderer $this */
+$account = $this->auth()->getManager();
+/** @var \VuFind\Search\SearchOrigin\AbstractSearchOrigin|null $searchOrigin */
+$searchOrigin = $this->get('searchOrigin');
```

Comment:
It might be more clear to use the layout() helper here instead of $this->get for consistency with other code that fetches variables from the layout container.